src/D2L.Bmx/Okta/State/OktaAuthenticatedState.cs

```diff
@@ -2,5 +2,5 @@ namespace D2L.Bmx.Okta.State;
 
 internal record OktaAuthenticatedState(
 	bool SuccessfulAuthentication,
-	string OktaSessionToken
+	string OktaSessionId
```

Comment:
Think it's better to be passing around a session id instead. This value is currently what gets stored in session file